io/serialport/25-serial.js

```diff
@@ -121,6 +135,20 @@ module.exports = function(RED) {
             node.port = serialPool.get(this.serialConfig);
             // Serial Out
             node.on("input",function(msg) {
+                if (msg.hasOwnProperty("baudrate")) {
+					var baud = parseInt(msg.baudrate);
+					if (isNaN(baud)) {
+						var errmsg = err.toString().replace("Serialport","Serialport "+node.port.serial.path);
+						node.error(errmsg,"Cannot parse baudrate property");
```

Comment:
as per above errmsg should be the text, 2nd param should just be msg.
It should also be translatable - the text should be in the locale file - eg see line 61 etc.

Reply:
Changed like line 161.